tests/test_legendastv.py

```diff
@@ -0,0 +1,321 @@
+        'Man.of.Steel.2013.R6.SCR.TS.Audio.Xvid.mp3-CRYS.srt (a50ef54a73c490f3d7f63c333f5d3e07)',
+        'Man.of.Steel.2013.TS.LiNE.READNFO.x264.HiGH.srt (a50ef54a73c490f3d7f63c333f5d3e07)',
+        'Man of Steel (2013) 720p R6 LiNE 900MB Ganool.srt (a50ef54a73c490f3d7f63c333f5d3e07)',
+        'Man of Steel 2013 R6 TS XViD-PLAYNOW .srt (a50ef54a73c490f3d7f63c333f5d3e07)',
+        'MAN OF STEEL 2013 R6-1080P AC3 MURDER.srt (a50ef54a73c490f3d7f63c333f5d3e07)',
+        'Man of Steel 2013  720P TS XviD MP3 MiLLENiUM.srt (a50ef54a73c490f3d7f63c333f5d3e07)',
+        'Man.of.Steel.2013.720p.R6.LiNE.x264.AAC-DiGiTAL.srt (a50ef54a73c490f3d7f63c333f5d3e07)',
+        'Man.of.Steel.TS.XviD-WAR.srt (a50ef54a73c490f3d7f63c333f5d3e07)',
+        'Man Of Steel 2013 REPACK TS XViD UNiQUE (SilverTorrent).srt (a50ef54a73c490f3d7f63c333f5d3e07)',
+        'Man.Of.Steel.2013.720p.BluRay.x264-Felony.srt (525e738d4866b)',
+        'Man.Of.Steel.2013.480p.BluRay.x264-mSD.srt (525e738d4866b)',
+        'Man.Of.Steel.2013.480p.BRRip.Xvid.AC3-UNDERCOVER.srt (525e738d4866b)',
+        'Man.of.Steel.2013.720p.BluRay.x264.YIFY.srt (525e738d4866b)',
+        'Man.Of.Steel.2013.720p.BRRip.x264.AC3-EVO.srt (525e738d4866b)',
+        'Man.Of.Steel.2013.BDRip.x264-4PlayHD.srt (525e738d4866b)',
+        'Man.Of.Steel.2013.BDRip.x264-Larceny.srt (525e738d4866b)',
+        'Man.Of.Steel.2013.BDRiP.XViD-NoGRP.srt (525e738d4866b)',
+        'Man of Steel [2013] BRRip XViD [AC3]-ETRG.srt (525e738d4866b)',
+        'Man Of Steel 2013 720p BRRiP XViD UNiQUE.srt (525e738d4866b)',
+        'Man Of Steel 2013 BRRip x264 AC3 UNiQUE.srt (525e738d4866b)',
+        'man.of.steel.2013.1080p.bluray.x264-sector7.srt (526076d4af488)',
+        'Man.Of.Steel.2013.720p.BRRip.x264-YIFY.srt (525dd8547cb72)',
+        'Man.Of.Steel.2013.BDRip.x264-Larceny.srt (525d86f6c6560)',
+        'Man.Of.Steel.2013.720p.BluRay.x264-Felony.srt (525d86f6c6560)',
+        'Man.Of.Steel.2013.720p.BRRip.x264.AAC-ViSiON.srt (5262e21d58bab)',
+        'Man.Of.Steel.2013.1080p.BluRay.x264.anoXmous_.srt (52cc3e195127e)',
+        'Man.Of.Steel.3D.2013.1080p.BluRay.Half-SBS.DTS.x264-PublicHD.srt (527a1eda17867)',
+        'Man of Steel 2013 1080p Blu-ray Remux AVC DTS-HD MA 7.1 - KRaLiMaRKo.srt (52604f2d2099a)',
+        'Man of Steel 2013 1080p Blu-ray 3D Remux.srt (5285f5daac692)',
+        'Man of Steel 2013 1080p Blu-ray 3D Remux.Forced.srt (5285f5daac692)',
+        'Man of Steel (BDRip.x264-Larceny Subrip).srt (527d55c0c02bc)',
+        'Man of Steel (2013) DVDRip XviD-MAXSPEED.srt (54edd551e869a)'
+    }
+
+    with LegendasTvProvider(USERNAME, PASSWORD) as provider:
+        provider.initialize()
+        subtitles = provider.query(language, title=video.title, year=video.year)
+        provider.terminate()
+
+    assert {'%s (%s)' % (subtitle.name, subtitle.subtitle_id) for subtitle in subtitles} == expected_subtitles
```

Comment:
You can use tuples here, it's hashable:

    ('Superman Man Of Steel 2013 ANOTHER NEW SOURCE TS XViD UNiQUE.srt', 'a50ef54a73c490f3d7f63c333f5d3e07')